### PR TITLE
L1 Norm Implementation

### LINKED ISSUE #182: L1 error calculation
The current version only calculates L2 norm. The L1 norm should be added

### PR DESCRIPTION
**Pull request by sunmJason**
L1 norm error calculation has been added to SoluitonErrorMonitor and also included in one of the integrationTests.

This closes #182

### CODE REVIEW

File: tests/integrationTests/inputs/compressibleCouetteFlow.yaml [hunk 1]
```diff
@@ -71,16 +71,29 @@ solver: !ablate::finiteVolume::CompressibleFlowSolver
         fieldName: euler
         field: "1.0, 216500.0, 50.0, 0.0"
   monitors:
+    - !ablate::monitors::SolutionErrorMonitor
```
Comment (mmcgurn) [inline tests/integrationTests/inputs/compressibleCouetteFlow.yaml:74]: Did you update the outputs file to account for these changes?

**Review by mmcgurn — CHANGES_REQUESTED** [review]
@sunmJason Thanks for the PR!  It doesn't look like you bumped the version number.  Do you know if the tests pass locally?

File: tests/integrationTests/inputs/compressibleCouetteFlow.yaml (hunk 1 above)
Comment (sunmJason) [reply tests/integrationTests/inputs/compressibleCouetteFlow.yaml:74]: I have run the integration test yaml and the tests passed locally. I am not sure what you meant by updating the output file.

Comment (mmcgurn) [reply tests/integrationTests/inputs/compressibleCouetteFlow.yaml:74]: Can you try running them again?  When I checkout your code and run the integration tests they fail.  You will need to update the corresponding expected output in `tests/integrationTests/outputs/compressibleCouetteFlow.txt`

`
/Users/mcgurn/scratch/ablate/tests/testingResources/MpiTestFixture.cpp:150: Failure
Expected equality of these values:
  expectedLine
    Which is: "Timestep: 0000 time = 0        \t l2_norm error:"
  actualLine
    Which is: "Timestep: 0000 time = 0        \t l1_norm error:"
 on line Timestep: 0000 time = 0        	 l2_norm error: of file outputs/compressibleCouetteFlow.txt
Generated output file: Tests_IntegrationTestsSpecifier.ShouldRun_inputs_compressibleCouetteFlow_yaml.txt
`

Comment (sunmJason) [reply tests/integrationTests/inputs/compressibleCouetteFlow.yaml:74]: I see what you mean. I have changed the output .txt to match what I see in the printout outputs. I remember that I can run individual test from the Clion UI. How can I do that? Right now I am running All CTest.

**Review by mmcgurn — APPROVED** [review]

**Pull request merged**